proto/tendermint/abcix/types.proto

```diff
@@ -250,8 +256,16 @@ message ResponseDeliverBlock {
 }
 
 message ResponseCheckBlock {
-  uint32 code = 1;
-  string codespace = 2;
+  uint32                            code                    = 1;
+  bytes                             result_hash             = 2;
```

Comment:
result hash can be derived from `deliver_txs` and `events`, probably no need to include in the response